var/spack/repos/builtin/packages/py-pybind11/package.py

```diff
@@ -64,3 +66,8 @@ def cmake_args(self):
                 'ON' if self.run_tests else 'OFF')
         ]
         return args
+
+    def install(self, spec, prefix):
```

Comment:
can you remove this function and post the error that you are seeing (as `super(PyPybind11, self).install(spec, prefix)` should literally just call `python('setup.py', 'install', '--prefix={0}'.format(prefix))`

Reply:
It will not call python(...) since py-pybind11 is derived from CMakePackage. The error I am seeing when installing py-projectq is that when running "python setup.py" the module import of pybind11 fails.

Reply:
First sentence might seem a little harsh, in my opinion it will not be called. Sorry for that!

Reply:
Argh, no you are perfectly right. I just (blindly) assumed that `py-pybind11` derived from `PythonPackage`. So yes, you will need to do that call, or update the derived package.

Now I'm wondering how it worked before, I'll probably have to dig into that, maybe I can get to that tomorrow

Reply:
In my opinion ir did not work before. All other packages that use pybind11 use cmake for building and seem to be in no need of the python files.